src/OMSimulatorLib/Component.cpp

```diff
@@ -47,3 +47,12 @@ oms_status_enu_t oms3::Component::exportToSSD(pugi::xml_node& node) const
   node.append_attribute("name") = this->getName().c_str();
   return oms_status_ok;
 }
+
+oms3::Connector *oms3::Component::getConnector(const oms3::ComRef &cref)
+{
+  for(int i=0; element.getConnectors() && element.getConnectors()[i]; ++i) {
+    if(element.getConnectors()[i]->getName() == cref)
+      return element.getConnectors()[i];
```

Comment:
I think we should have a `std::vector` for the connectors and just link `element.connectors` to it.